### ISSUE TITLE
can I use https using flask-ngrok

### ISSUE THREAD

**Issue by Arunpoochelvan** [gstaff/flask-ngrok#21]

**Comment by shuxiongwuziqi** [comment]
Yes, of course! Since ngrok provide http and https service by default. Just change 'http' into 'https', the website will work well. And I saw the code, and find author replace 'https' with 'http' in here [code](https://github.com/gstaff/flask-ngrok/blob/master/flask_ngrok.py#L43), I don't know why do this.

**Comment by gitfabianmeyer** [comment]
Any further explanation on this? I cannot make it run with https

**Comment by russorat** [comment]
PM from ngrok here. I also don't know why the author decided to replace https with http in this line: https://github.com/gstaff/flask-ngrok/blob/33a2e70898c4fb72b645715dacd8789769a99350/flask_ngrok.py#L43

Whatever link this library spits out, you should use the https version of the url instead. That solves the issue with the browser warning interstitial as well.
